packages/contracts-bedrock/hardhat.config.ts

```diff
@@ -84,6 +84,12 @@ const config: HardhatUserConfig = {
       accounts: [process.env.PRIVATE_KEY_DEPLOYER || ethers.constants.HashZero],
       live: true,
     },
+    'getting-started': {
+      chainId: 5,
+      url: process.env.L1_RPC || '',
+      accounts: [process.env.PRIVATE_KEY_DEPLOYER || ethers.constants.HashZero],
+      live: true,
+    },
```

Comment:
What is this? 
Where will it be documented?
Should it be on goerli?

Reply:
It's the network that will be used for the OP Stack documentation tutorial. It should be on Goerli, yes.